README.rst

```diff
@@ -223,6 +223,98 @@ Run tests with:
 
     ./runtests.py
 
+
+The django app `tests/openwisp2/sample_firmware_upgrader.py` adds some changes on
+top of the `openwisp-firmware-upgrader` module with the sole purpose of testing the
+module's extensibility.
+
+Extending openwisp-firmware-upgrader
+---------------------
+
+The `tests/openwisp2/sample-firmware-upgrader` may serve as an example for
+extending *openwisp-firmware-upgrader* in your own application.
+
+*openwisp-firmware-upgrader* provides a set of models and admin classes which can
+be imported, extended and reused by third party apps.
+
+To extend *openwisp-firmware-upgrader*, **you MUST NOT** add it to ``settings.INSTALLED_APPS``,
+but you must create your own app (which goes into ``settings.INSTALLED_APPS``), import the
+base classes from openwisp-firmware-upgrader and add your customizations.
+
+In order to help django find the static files and templates of *openwisp-firmware-upgrader*,
+you need to perform the steps described below.
+
+1. Install ``openwisp-utils``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Install (and add to the requirement of your project) `openwisp-utils
+<https://github.com/openwisp/openwisp-utils>`_::
+
+    pip install openwisp-utils
+
+2. Add ``EXTENDED_APPS``
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Add the following to your ``settings.py``:
+
+.. code-block:: python
+
+    EXTENDED_APPS = ('openwisp-firmware-upgrader',)
```

Comment:
this doesn't look right, I think it should be as: https://github.com/openwisp/openwisp-firmware-upgrader/pull/9/files#diff-9dec27df1ed1039b50d2b96520f1de08R199

Reply:
Yeah, my bad! :smile:
Thanks for fixing it! :smile: